### PR TITLE
complesso.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119839

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-25 в 18 23 18

Снимок экрана 2022-05-25 в 18 23 34

</details><br/>

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -55,11 +55,12 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
...
...isti.com,fox47.com,foxkansas.com,foxrichmond.com,gazetaby.com,gercekizmir.com,gidavitrini.com.tr,govoritmoskva.ru,halklailiskiler.com.tr,hentairules.net,hizlifilmizlehd.com,hornygamer.com,inforesist.org,jeotermalhaberler.com,jetvis.ru,kachelmannwetter.com,karsolay.com,kbsi23.com,kdsm17.com,kevinpurcell.org,kg-portal.ru,kinopoisk.userecho.com,kp.ua,larende.com,losst.ru,magazinekibi.com,manisamanset.gen.tr,mforos.com,mogilev.online,muchohentai.com,multifilemirror.com,mvs.gov.ua,mytuner-radio.com,mywebtimes.com,namemc.com,news-journalonline.com,nk.org.ua,nkj.ru,noticierodigital.com,ntp-servers.net,obltv.ru,orbita.dn.ua,pctip.ru,podrobnosti.ua,ppt.ru,proglang.su,raider.io,raleighcw.com,real-vin.com,rmj.ru,rusvesna.su,sanatatak.com,seirsanduk.us,simdifilmizle.com,sonkaleizmir.com,sp-mobile-blog.com,star64.tv,tayga.info,tomiks33.ru,torrentbe.net,torrentrg.org,tqm.com.ua,tribundergi.com,tva.in.ua,utofiles.com,washingtontimes.com,wcti12.com,womanel.com.ua,wpgh53.com,wtto21.com,wutv29.com,xn----8sbcm1bmcar0m.xn--p1ai,gunafrica.co.za,opentrackers.org,tv.ua,ewdn.com,meanjin.com.au,openworldnews.net,radiko.jp,anime.dmkt-sp.jp,bushikaku.net,toyokeizai.net,buzzap.jp,tsubasa.im##.twitter-timeline
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:59]: Also add `complesso.jp` to the rule.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -40,6 +40,7 @@ johndoeblog.org,aviationwire.jp,getnavi.jp,entamerush.jp,gourmetpress.net,career
 !
 !
 thetennisdaily.jp##.side-bnr-area
+complesso.jp###custom_html-22
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:43]: Not an ad and there is no need due to `##.twitter-timeline`

File: SocialFilter/sections/specific.txt [hunk 3]
```diff
@@ -55,11 +55,12 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
...
 !
 lycoris-recoil.com##.l-header_share
 lycoris-recoil.com##.p-timeline
+complesso.jp##iframe[id^="twitter-widget"]
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:63]: I don't see the need for that.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — CHANGES_REQUESTED** [review]
Missed leftover `complesso.jp###custom_html-7`

File: JapaneseFilter/sections/css_extended.txt [hunk 4]
```diff
@@ -455,7 +455,7 @@ clublucifer.net#?#.blockWrapper div[id^="mhub-"]:upward(.blockWrapper)
 news.dwango.jp#?#.bnr-s_2 > div[id^="div-gpt-ad"]:upward(1)
 kanaloco.jp#?##recommendations > li > div[class^="ad"]:upward(1)
 detail.chiebukuro.yahoo.co.jp#?#aside[class^="ClapLv3SubList_Chie-SubList__ListItem__"] > div.infdAd:upward(1)
-nilcollection.net,fragrantica-japan.com,taisy0.com,gadgerba.com#?#.cardtype__article > ins.adsbygoogle:upward(1)
+complesso.jp,nilcollection.net,fragrantica-japan.com,taisy0.com,gadgerba.com#?#.cardtype__article > ins.adsbygoogle:upward(1)
```
[the remark this answers is not in the document]
Comment (zloyden) [reply JapaneseFilter/sections/css_extended.txt:458]: Please provide the screenshot with the element code.

[the remark this answers is not in the document]
Comment (jellizaveta) [reply JapaneseFilter/sections/css_extended.txt:458]: <details><summary>Screenshot:</summary>

Снимок экрана 2022-05-26 в 13 56 34

</details><br/>

**Review by zloyden — APPROVED** [review]

**Pull request merged**